docs/docusaurus/docs/tutorials/quickstart/quickstart.md

```diff
@@ -30,6 +31,55 @@ Windows support for the open source Python version of GX is currently unavailabl
 - pip
 - An internet browser
 
+## Quickstart Workflow
+
+This quickstart walks you through the steps required to implement a simple, end-to-end GX workflow.
```

Comment:
```suggestion
This quickstart walks you through the steps required to implement a simple, end-to-end GX workflow.
```
This topic already contains an introductory statement. This statement should introduce the diagram and its purpose. For example:

The following diagram illustrates the GX  data validation workflow. 

I would suggest incorporating this diagram into the overview or adding it before the **Prerequisites** section.